src/main/java/sg/ncl/MainController.java

```diff
@@ -1929,6 +1929,20 @@ public String stopExperiment(@PathVariable String teamName, @PathVariable String
         return abc(teamName, expId, redirectAttributes, realization, request);
     }
 
+    @RequestMapping("/get_topology/{teamName}/{expId}")
+    @ResponseBody
+    public String getTopology(@PathVariable String teamName, @PathVariable String expId) {
+        try {
+            HttpEntity<String> request = createHttpEntityHeaderOnly();
+            ResponseEntity response = restTemplate.exchange(properties.getTopology(teamName, expId), HttpMethod.GET, request, String.class);
+            log.info("experiment topo: {}", response.getBody().toString());
+            return "data:image/png;base64," + response.getBody();
+        } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)